kernel/src/svsm.rs

```diff
@@ -441,6 +443,11 @@ pub extern "C" fn svsm_main() {
         prepare_fw_launch(fw_meta).expect("Failed to setup guest VMSA/CAA");
     }
 
+    #[cfg(feature = "mstpm")]
+    if let Err(e) = vtpm_init() {
+        panic!("vTPM failed to initialize - {:?}", e);
+    }
```

Comment:
This will also display the error and is shorter:

```suggestion
    vtpm_init().expect("vTPM failed to initialize");
```

Reply:
I will do that, thanks.